task/backend/scheduler/scheduler.go

```diff
@@ -0,0 +1,78 @@
+package scheduler
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/influxdb"
+)
+
+type Storage interface {
+	// UpdateLatestCompleted will allow us to set a latest completed time for an id.
+	UpdateLatestCompleted(ctx context.Context, id influxdb.ID, lastRun time.Time) error
+
+	// retrieve the latest completed for this schedulable id.
+	// If GetLatestCompleted is empty scheduler will use "now" as its latest completed
+	GetLatestCompleted(ctx context.Context, id influxdb.ID) (time.Time, error)
```

Comment:
Comment should be in the form // GetLatestCompleted ...

Reply:
What do you think of changing `GetLatestCompleted` to `CompletionTime` or `LatestTime` or `LatestCompletionTime` ?  I think it is good to have units and that generally no need for a `Get` prefix.

Reply:
I like `LatestCompletionTime`